### PR TITLE
copy-link-style-changes2

### PR DESCRIPTION
**Pull request by yeelali14**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=ey1agO8)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,3 +74,17 @@ And in the free fall I will realize
...
+And in the free fall I will realize
+I'm better off when I hit the bottom
+
+Lady gaga the queen
+
+And in the free fall I will realize
+I'm better off when I hit the bottom
+
+Lady gaga the queen
+And in the free fall I will realize
```
Comment (paparoach14) [inline README.md:87]: fsdf

**Review by paparoach14 — CHANGES_REQUESTED** [review]
nb

**Review by EladKohavi — APPROVED** [review]
fsda

**Review by EladKohavi — APPROVED** [review]
jkhfjkgfjghf

**Review by EladKohavi — APPROVED** [review]
gdfgfd

**Pull request closed without merging**